src/applications/mhv/landing-page/tests/actions/hca-es.unit.spec.js

```diff
@@ -0,0 +1,107 @@
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+import environment from '~/platform/utilities/environment';
+import {
+  FETCH_HCA_ENROLLMENT_STATUS_FAILED,
+  FETCH_HCA_ENROLLMENT_STATUS_STARTED,
+  FETCH_HCA_ENROLLMENT_STATUS_SUCCEEDED,
+  fetchHcaEnrollmentStatus,
+  fetchHcaEnrollmentStatusFailed,
+  fetchHcaEnrollmentStatusStarted,
+  fetchHcaEnrollmentStatusSucceeded,
+} from '../../actions/hca-es';
+import { initialState } from '../../reducers/hca-es';
+
+const enrollmentStatus = {
+  effectiveDate: '2019-01-02T21:58:55.000-06:00',
+  priorityGroup: 'Group 8G',
+};
+
+const mockApi = (path, data, status = 200) => {
+  const url = `${environment.API_URL}${path}`;
+  const server = setupServer(
+    rest.get(url, (_, res, ctx) => res(ctx.status(status), ctx.json(data))),
+  );
+  server.listen();
+  return server;
+};
+
+describe('HCA Enrollment Status actions', () => {
+  describe('fetchHcaEnrollmentStatusStarted', () => {
+    it('returns FETCH_HCA_ENROLLMENT_STATUS_STARTED action', () => {
+      const { type } = fetchHcaEnrollmentStatusStarted();
+      expect(type).to.equal(FETCH_HCA_ENROLLMENT_STATUS_STARTED);
+    });
+  });
+
+  describe('fetchHcaEnrollmentStatusFailed', () => {
+    it('returns FETCH_HCA_ENROLLMENT_STATUS_FAILED action', () => {
+      const message = { errorMessage: 'it broke' };
+      const { type, payload } = fetchHcaEnrollmentStatusFailed(message);
+      expect(type).to.equal(FETCH_HCA_ENROLLMENT_STATUS_FAILED);
+      expect(payload).to.equal(message);
+    });
+  });
+
+  describe('fetchHcaEnrollmentStatusSucceeded', () => {
+    it('returns FETCH_HCA_ENROLLMENT_STATUS_SUCCEEDED action with data', () => {
+      const { type, payload } = fetchHcaEnrollmentStatusSucceeded(
+        enrollmentStatus,
+      );
+      expect(payload).to.equal(enrollmentStatus);
+      expect(type).to.equal(FETCH_HCA_ENROLLMENT_STATUS_SUCCEEDED);
+    });
+  });
+
+  describe('fetchHcaEnrollmentStatus', () => {
+    it('gets the enrollment status', done => {
+      const path = '/v0/health_care_applications/enrollment_status';
+      const server = mockApi(path, enrollmentStatus);
```

Comment:
CONSIDERATION: I typically like to do API testing in e2e tests rather than unit tests.  Since this is testing an interaction with an external service, it always felt cleaner

Not a blocker